Update whitelist.md (#25)
SIsilicon/WorldEdit-BE-Docs: docs/usage/whitelist.md

# Whitelisting

By default, the add-on has a whitelist enabled. This means that players need explicit permission to use its features. You can enable and disable this whitelist with `/function wedit/whitelist/on` and `/function wedit/whitelist/off` respectively.

> 

Reviewer: For content like this, it would be better to put it in a block. like this. be sure to preview your work before making more changes

https://github.com/SIsilicon/WorldEdit-BE-Docs/blob/6bdffbce85791e05c189c78826019f9af9b8599f/docs/commands.md?plain=1#L15

> ### IMPORTANT
> this will also fully turn off the permision system of the addon meaning everyone can use any worldedit command 

Reviewer 2: Haven't made a suggestion using this feature before. Let's see how it goes :D

## Admonition Syntax

Adding on to @SIsilicon's comment:
```suggestion
!!! warning

    This will also fully turn off the permission system of the addon, meaning anyone will be able to use any WorldEdit command.

```
See the documentation for that admonition syntax [here](https://python-markdown.github.io/extensions/admonition/).

Material for MkDocs also has documentation for it, but note that the output won't look the same: see [here](https://squidfunk.github.io/mkdocs-material/reference/admonitions).

## Spacing

Note that it's really important to put **4** spaces before the paragraph, otherwise the admonition won't render correctly:
### 2 Spaces
```
!!! warning

  This will also fully turn off the permission system of the addon, meaning anyone will be able to use any WorldEdit command.
```
Renders as
<img width="720" height="133" alt="image" src="https://github.com/user-attachments/assets/3b33c2d3-1f54-4184-b547-42c0e8b6c569" />

### 4 Spaces

```
!!! warning

    This will also fully turn off the permission system of the addon, meaning anyone will be able to use any WorldEdit command.
```
Renders as 
<img width="720" height="133" alt="image" src="https://github.com/user-attachments/assets/773bea66-2b73-4c31-b1ab-bd261e59894a" />
